src/fontra/client/web-components/glyphs-search.js

```diff
@@ -68,8 +68,8 @@ export class GlyphsSearch extends UnlitElement {
       { key: "glyphName", title: "glyph name", width: "10em", isIdentifierKey: true },
       {
         key: "unicode",
-        width: "5em",
-        get: (item) => makeUPlusStringFromCodePoint(item.unicodes[0]),
+        width: "fit-content",
+        get: (item) => item.unicodes?.map(makeUPlusStringFromCodePoint).join(","),
```

Comment:
Let's remove the `?`: I just checked, and `.unicodes` comes straight from the `glyphMap` object, where values _must_ be arrays (but can be empty).
```suggestion
        get: (item) => item.unicodes.map(makeUPlusStringFromCodePoint).join(","),
```